tests/misc/default_board_config/src/main.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2019 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+
+#ifdef CONFIG_IEEE802154
+#error IEEE802154 should not be part of default board configuration
+#endif
+
+#ifdef CONFIG_ADC
+#error ADC should not be part of default board configuration
+#endif
+
+#ifdef CONFIG_AUDIO
+#error AUDIO should not be part of default board configuration
+#endif
+
+#ifdef CONFIG_BT
+#error BT should not be part of default board configuration
+#endif
```

Comment:
I agree that BT should be disabled, but this is not what the guidelines say, is it?

> Provide pin and driver configuration that matches the board’s valuable components such as sensors, buttons or LEDs, and communication interfaces such as USB, Ethernet connector, or Bluetooth/Wi-Fi chip.

Reply:
https://docs.zephyrproject.org/latest/guides/porting/board_porting.html

Reply:
This paragraph needs to be clarified.
This should actually be configured, but not enabled.

Reply:
I see